fixture-model-registry/code/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "experiments-get-started",
```

Comment:
should be different?

should we install DVC extension also?

why do we need codespaces for this btw?